Formula/xbyak.rb

```diff
@@ -0,0 +1,43 @@
+class Xbyak < Formula
+  desc "C++ JIT assembler for x86 (IA32), x64 (AMD64, x86-64)"
+  homepage "https://github.com/herumi/xbyak"
+  url "https://github.com/herumi/xbyak/archive/refs/tags/v6.69.1.tar.gz"
+  sha256 "40b41df141000688d7d629cc9f4bc313f3adfa6be0c42c520064608ed11a93fc"
+  license "BSD-3-Clause"
+  head "https://github.com/herumi/xbyak.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
Is this needed?

Reply:
yes, otherwise livecheck parses weird xbyak tags:
```
% brew livecheck --debug xbyak
/opt/homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /opt/homebrew/Library/Taps/ilya-lavrenov/homebrew-core/Formula/xbyak.rb

Formula:          xbyak
Livecheckable?:   No

URL:              https://github.com/herumi/xbyak/archive/refs/tags/v6.69.1.tar.gz
URL (processed):  https://github.com/herumi/xbyak.git
Strategy:         Git

Matched Versions:
2-released, 20131205, 20170401, 3.71, 4.70, 4.71, 4.82, 4.83, 4.84, 4.85, 4.86, 4.87, 4.88, 4.89, 4.90, 4.901, 5.00, 5.03, 5.10, 5.11, 5.31, 5.33, 5.40, 5.41, 5.42, 5.43, 5.431, 5.432, 5.50, 5.51, 5.52, 5.53, 5.60, 5.601, 5.61, 5.62, 5.63, 5.631, 5.64, 5.65, 5.66, 5.661, 5.67, 5.68, 5.70, 5.71, 5.72, 5.73, 5.74, 5.75, 5.751, 5.77, 5.78, 5.79, 5.80, 5.801, 5.802, 5.81, 5.82, 5.83, 5.84, 5.85, 5.86, 5.87, 5.88, 5.89, 5.891, 5.90, 5.91, 5.911, 5.92, 5.93, 5.94, 5.941, 5.95, 5.96, 5.97, 5.98, 5.99, 5.991, 5.992, 5.993, 5.994, 5.995, 5.996, 5.997, 6.00, 6.01, 6.02, 6.03, 6.04, 6.041, 6.05, 6.051, 6.052, 6.06, 6.60, 6.60.1, 6.60.2, 6.61, 6.61.1, 6.61.2, 6.62, 6.63, 6.64, 6.65, 6.66, 6.67, 6.68, 6.69, 6.69.1, 20150301

xbyak: 6.69.1 ==> 20170401
```

Reply:
yeah, it is neded.